projects/ad405x_iio/scripts/ad405x.py

```diff
@@ -0,0 +1,213 @@
+# Copyright (C) 2023 Analog Devices, Inc.
```

Comment:
Any reason why we haven't pushed this file to the pyadi repo?

Reply:
There's already a PR for adding the pyadi drivers and examples to their GitHub repo https://github.com/analogdevicesinc/pyadi-iio/pull/565